image_client/PIC_dbcreate/Picturae_DDL.sql

```diff
@@ -0,0 +1,39 @@
+# ddl to run inside docker container / or vm for creation of new picturae upload batch database
```

Comment:
remove database names. Always and forever. :-)

Reply:
note: it's never required (you can specify the db name on the command line of the mysql statement, as you do below in the docker), so there's no upside, but there's a downside - you can't use it elsewhere without changing this file.